### DISCUSSION TITLE
News scroll doesn't work

### DISCUSSION THREAD

**Discussion by rathilakshika** [alshedivat/al-folio#2620] [Q&A] [answered]
I am trying to use the vertical scrollbar for the news items. I set the "scrollable" to true in the _config.yml file - https://github.com/rathilakshika/rathilakshika.github.io, but that doesn't work. Is there some other parameter to set as well?

**Comment by george-gca** [comment, marked as answer]
Somehow the height limit is too high, that's why it is not triggering the scroll bar. Try reducing it here:

https://github.com/alshedivat/al-folio/blob/1d4ce5a313d1c41e73c843587692eabebad96e00/_includes/news.liquid#L7
